src/modules/AuthenticationModule/components/Verify Account/VerifyAccount.tsx

```diff
@@ -1,8 +1,103 @@
 
+import { useContext } from "react";
+import logo from "../../../../assets/PMS 3.png";
+import { APIcontext } from "../../../../Context/baseUrl";
+import { useForm } from "react-hook-form";
+import axios from "axios";
+import { useNavigate } from "react-router-dom";
+import { ToastContext } from "../../../../Context/ToastContext";
+
 const VerifyAccount = () => {
+  const {baseUrl}=useContext(APIcontext)
+  const {getToastValue}=useContext(ToastContext)
+
+  const navigate = useNavigate();
+
+  const {
+      register,
+      handleSubmit,
+      formState: { errors },
+  } = useForm();
+  const onSubmit = async (data) => {
+    try {
+        let response = await axios.put(
+            `${baseUrl}/Users/verify`,data
+        );
+        getToastValue('success','Verification success you can login now')
```

Comment:
We need to provide the backend message first and if there was no message show the fallback message instead.